sanitizers/app/src/asan/resources/lib/arm64-v8a/wrap.sh

```diff
@@ -0,0 +1,31 @@
+#!/system/bin/sh
+# Copied from https://android.googlesource.com/platform/ndk/+/refs/heads/master/wrap.sh/asan.sh
```

Comment:
Bonus points if you work out the gradle-fu to copy this from the NDK so it doesn't get stale (every so often we need to extend the jdwp workarounds to account for an ART change)

Reply:
Done.